### ISSUE TITLE
Fetch content from database using json instead of setting it directly in PHP

### ISSUE THREAD

**Issue by halvors** [InfectedLan/InfectedAPI#75]
Fetch content using json as well. We cab avoid refreshing of pages when someone click on a button, removes a member etc.

What do you think?

**Comment by halvors** [comment]
1. If JS crashes, the user wouldn't be able to post any forms at all anyway.
2. If php crashes, we're all doomed anyway.
3. This would actually benefit users with slow connected, as content updates wouldn't require them to download the whole page again, especially images :)

**Comment by halvors** [comment]
http://www.smashingmagazine.com/2012/02/09/beginners-guide-jquery-based-json-api-clients/

**Comment by petterroea** [comment]
I like the idea. Let me have a look at how doable it is.

It would not be first priority as it requires a remake of a lot of stuff, but i like the idea.

**Comment by halvors** [comment]
I've made the branch "fetch-json" that could be our playground for this until we reach a stable state it will not be merged into mainline development.

**Comment by petterroea** [comment]
Didn't we change our mind and decide to go with templates instead?

**Comment by halvors** [comment]
From my point of view, templates would basically be the same thing right?

**Comment by petterroea** [comment]
No, what templates does is that instead of having php and html mixed, we have variable-like tags in the html that are replaced with data on runtime. Its 10/10

**Comment by halvors** [comment]
@petterroea Any progress on this?

**Comment by petterroea** [comment]
The test api and exams has been in the way. Will have a look at it soon, but i want travis working first
